### PR TITLE
Publish the build for decentraland website as an npm package

### PR DESCRIPTION
**Pull request by jmoguilevsky**
# What? 
...

# Why? 
...

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -223,7 +223,7 @@ jobs:
           paths:
             - website/build
 
-  publish-ecs:
+  publish-ecs-and-website:
```
Comment (menduz) [inline .circleci/config.yml]:
```suggestion
  publish-ecs:
```

File: website/scripts/publish.sh [hunk 2]
```diff
@@ -2,6 +2,4 @@
 
 cp static/package.json build/
 cd build
-echo $(ls ../../kernel/scripts)
-npx tsc --build ../../kernel/scripts/tsconfig.json
-node ../../kernel/scripts/npmPublish.js
+npx -T ../scripts/npmPublish.ts
```
Comment (menduz) [inline website/scripts/publish.sh]: What does `npx -T` do?

Comment (jmoguilevsky) [reply website/scripts/publish.sh]: It does nothing on it's own, it was a typo, already sent the fix on another commit

Comment (jmoguilevsky) [reply website/scripts/publish.sh]: It was `npx ts-node -T ...` so that ts-node doesn't validate typescript errors and just transpiles the code

**Pull request closed without merging**